repositories.yaml

```diff
@@ -834,40 +834,20 @@ applications:
       - glean/metrics.yaml
     ping_files:
       - glean/pings.yaml
-    dependencies:
-      - glean-js
+    dependencies: []  # Empty. Dependencies are set per channel.
     moz_pipeline_metadata_defaults:
       expiration_policy:
         delete_after_days: 180
     channels:
       - v1_name: mozilla-vpn
         app_id: mozillavpn
-        app_channel: release
```

Comment:
IMO it'd be better to leave both channels with `app_channel: release`, which will help clarify that these are both actually release channels (not beta/nightly).

Reply:
Good point. Was there a reason why you removed this in your initial PR, @badboy?

Reply:
I tried giving them _different_ channel names, that were not "release", "beta" or "nightly" and that failed, so I just removed it.
Having them both called "release" is the right choice though.

Reply:
I'll add release back then